src/main/java/com/groupthree/mancala/controllers/PlayerDashboardController.java

```diff
@@ -0,0 +1,19 @@
+package com.groupthree.mancala.controllers;
+
+import javafx.fxml.FXML;
+
```

Comment:
Kindly add documentation to the controller classes. We're supposed to use the Java doc format. you can refer to Java Doc slides on canvas to see a sample format.

Reply:
@mofe64, Thanks for the review. Treated